portals/README.md

```diff
@@ -1,65 +1,6 @@
 # Community Portals
 
-Do you have a great portal that you would like to share? Please submit a pull request placing your portal in this folder. It'd be great to see what the community can come up with! Please also add yourself to this readme so your contribution is recognized.
```

Comment:
Looks like you accidentally removed a section.

Reply:
With this update, it looks like we can remove the `portals` folder at the root altogether. It's now empty. And then we can move that portion of the readme back to the main readme.

Reply:
@CodyTolene 
Do you need changes from my side or is this already resolved?

Reply:
Yep these suggestions are for your work here. It looks like you can remove the portals folder after your changes. It also looks like you removed a portion of the README that was unintended.

Reply:
The readme was just copied to the new folder along with all html files, no changes applied to the text. 

I also think the portals folder could be removed, but just for those who may be looking for it (many PR etc.), I placed a new readme file in the empty portals folder with a link to the new location of the html files & readme.